src/environment.js

```diff
@@ -147,8 +152,7 @@ var Environment = Obj.extend({
                 }
 
                 // Resolve name relative to parentName
-                if (parentName && (name.indexOf("./") == 0 ||
-                                   name.indexOf("../") == 0)) {
```

Comment:
Take a look at 4568ace8cfadbf3af9f954e1004c6acc16468863 and let me know what you think :)